src/server/request.py

```diff
@@ -22,7 +25,28 @@ def __init__(self, request, rtype=RequestType.AZURE):
 
     def _parse_cookies(self):
         if self._rtype == RequestType.CHERRY_PY:
-            self._cookies = self._request.cookie
+            # Google One Tap sends a cookie Python cannot parse
+            # https://stackoverflow.com/questions/66864766/one-tap-login-g-state-cookie-is-not-parseable-on-http-cookies-simplecookie
+            # Solution: remove and reparse
+            ### self._cookies = self._request.cookie
+            self._cookies = http.cookies.SimpleCookie()
+            http_cookie = self.get_header('Cookie')
+            if http_cookie:
+                # Remove any broken cookie which has XXX = { ... };
+                # Replace with XXX = ''
+                fixed_http_cookie = re.sub(r"\s*=\s*{.*\}\s*;", "='';", http_cookie)
+                try:
+                    self._cookies.load(fixed_http_cookie)
+                except http.cookies.CookieError as err:
+                    logging.info("Invalid cookie header received: %s", http_cookie)
+                # Check that our cookie survived parsing
+                if "shkola_session_id" in http_cookie and \
```

Comment:
better: if SESSION_COOKIE in http_cookie and SESSION_COOKIE not in self._cookies

SESSION_COOKIE can be imported from session.py

Reply:
Done